morango/models/fields/crypto.py

```diff
@@ -39,7 +50,13 @@
     CRYPTOGRAPHY_EXISTS = True
 except ImportError:
     CRYPTOGRAPHY_EXISTS = False
-
+except BaseException as e:
+    # Still catch PanicExceptions just in case.
+    if "Python API call failed" in str(e):
```

Comment:
Is this text specific to this exact error in this particular case? Did you run into some weird errors despite the above check not raising the `ImportError` that inspired adding this or are is this preemptive?

Not a blocker just got me thinking

Reply:
It will happen if any attempt to call a CPython API fails, I think - I could technically have not included this because of the pre-emptive checking I did above, but, if nothing else, this seemed useful to document the error we are trying to prevent.